packages/CLAPACK/patches/0001-hacks-for-WebAssembly-build.patch

```diff
@@ -75,114 +50,22 @@ index 57eff0d..d9a42da 100644
  	getenv_.o signal_.o s_stop.o s_paus.o system_.o cabs.o ctype.o\
```

Comment:
@rth what is the purpose of patching `MISC` in the line above? Is it another duplicate symbol issue?

Reply:
That's likely yes.

Reply:
Well it seems to work without the patch, so I'll leave it out